src/util/console-color.ts

```diff
@@ -24,7 +26,9 @@ export class ColoredConsole {
   constructor(public targetElement: HTMLElement) {}
 
   logs(): string {
-    return this.targetElement.innerText;
+    const logs = this.targetElement.cloneNode(true) as HTMLElement;
+    logs.querySelectorAll(".log-secret").forEach((e) => (e.innerHTML = ""));
```

Comment:
Why is this necessary ?

Reply:
When you used the download logs buttons then you get:
`BSSID: XX:XX:XX:XX:XX:XX[redacted]`
(I've redacted the actual BSSID the X's are an example)

This change is so that it downloads as:
`BSSID: [redacted]`

I've seen a few people share their logs this way we should protecting them 🙈